old/public/demos/breakout.html

```diff
@@ -11,30 +10,46 @@
     <div id="dailyContainer"></div>
 
     <script type="module">
+      import DailyIframe from "../extensions/esm-core.js";
```

Comment:
@chadbailey59 what's the plan for everything in the `old` folder? Is this going to get moved in an upcoming PR?

Reply:
Correct.